src/components/Nav.tsx

```diff
@@ -64,8 +79,12 @@ const NavItem = ({
 }: NavItemProp) => {
   const { title, url, subNav = [] } = item;
   const hasSubmenuItems = !!subNav.length;
+  const itemPath: URL | string = isUrlAbsolute(url)
+    ? removeDomainFromUrl(url)
+    : url;
+
   const [isNavItemActive, setNavItemActive] = useState(
-    currentPath === item.url
+    currentPath === itemPath
```

Comment:
We cannot remove the domain because we have more than a domain, so, if you remove the domain you are losing information, you do not know if the URL is from WebApp (`webappint.fromdoppler.net` or `app.fromdoppler.com`) or if it is from Doppler MVC (`appint.fromdoppler.net` or `app2.fromdoppler.com`).

In my opinion, you will need a temporal patch, like replacing `webappint.fromdoppler.net` by `testmenuint.fromdoppler.net`.

Reply:
The url with domain removed are only used to set an active item based on path. The link href property remains the same as  it comes in nav data.

But now that you mention losing information, this approach doesn't cover the case:  two links with the same path but diferent domain. It feels like a not possible scenario, but I can't confirm it.

Reply:
> But now that you mention losing information, this approach doesn't cover the case: two links with the same path but diferent domain. It feels like a not possible scenario, but I can't confirm it.

You are almost right. It is not a probable scenario, but it is certainly possible. And if it happens we will go crazy to find the bug.